use-cases/org-three-way-ecs.md

```diff
@@ -84,36 +90,35 @@ log archive account - s3, sns, sqs
     }
 -->
 
+### Configure AWS_PROFILE with an Organizational Administration Credentials
```

Comment:
can we maintain the numbering here? 1,2,3,... and subnumbering.

it comes handy when talking with tams and customers

Reply:
heading level should give the same information. Usually we don't include numbers in Headings

Reply:
do not agree on removing them, but however you prefer.